modules/dnn/test/test_layers.cpp

```diff
@@ -1448,4 +1448,93 @@ TEST(Layer_Test_Convolution, relu_fusion)
     normAssert(input, output);
 }
 
+// Check fuse convolution and relu wherein skip eltwise
+typedef testing::TestWithParam< tuple<Backend, Target> > Layers_Test_Subnet;
+TEST_P(Layers_Test_Subnet, convolutionx2_eltwise_scalex2_relu)
+{
+    Net net;
+    int id_conv[2];
+    int weightsShape[] = {1, 1, 1, 1};
+    {
+        LayerParams lp;
+        lp.set("kernel_size", 1);
+        lp.set("num_output", 1);
+        lp.set("bias_term", false);
+        lp.type = "Convolution";
+        lp.name = "testConv_1";
+
+        lp.blobs.push_back(Mat(4, &weightsShape[0], CV_32F, Scalar(-0.38)));
```

Comment:
No.
Each value of passed Mat should be different.

Test must be reliable. Including inputs.
Swapping of any of two different values on input should change result.

Reply:
@alalek corrected)